calico/felix/config.py

```diff
@@ -248,6 +248,16 @@ def __init__(self, config_path):
                            "spoofing their source IP.  (For example, "
                            "unprivileged containers.)",
                            False, value_is_bool=True)
+        self.add_parameter("ClusterSize", "Total cluster host count",
+                           0, value_is_int=True)
+        self.add_parameter("ClusterType", "Cluster type",
+                           "NA")
+        self.add_parameter("ClusterGUID", "Unique cluster identifier",
+                           "0")
+        self.add_parameter("VersionSignal",
```

Comment:
Also, new config parameters need to be be documented in the user-facing documentation.  That used to be `configuration.rst` in this repo but @eepyaich may have moved it to the new docs repo.